### DISCUSSION TITLE
Mini PC  - Desktop remoto

### DISCUSSION THREAD

**Discussion by Asky82** [indomus/forum#2837] [HUB: Home Assistant]
Ciao a tutti,

nuovo in questo mondo, dopo una giornata di duro lavoro :) sono riuscito finalmente ad installare debian 12 e HM su un nuovo mini PC.

Avrei due domande che immagino siano già state affrontate nel forum e chiedo scusa per questo, ma con GitHub non ho ancora ben capito come fare delle ricerche che portino i risultati aspettati. Se esiste una guida anche per questo.... :)

La mia idea è quella di posizionare il Mini PC in un rack senza monitor e nel momento del bisogno collegarmi via Windows Remote Desktop rdp.

Ora a banco con monitor collegato funziona tutto. Quello che non mi torna è:

- Se riavvio il Mini PC fin tanto che non eseguo il login non mi è possibile accedere in desktop remoto
- Senza schermo collegato idem come sopra( alternative software al HDMI plugin?

Grazie e buona domenica!

**Comment by Asky82** [comment]
Ho effettuato delle prove e per il momento ho risolto abilitando l'autologin and disabilitando il logoff. Immagino che non sia il meglio. 

Cosa ne pensate? C'è la possibilita di inserire il desktop remoto come servizio all'avvio?

Inizialmnetre installando tutto il sistema operativo ho seguito anche questa guida ma non so se è aggiornata perche io VNC non lo trovato già installato.

https://indomus.it/guide/come-collegarsi-via-ssh-e-vnc-a-linux-debian/

Grazie in anticipo per il vostro supporto.

**Reply by array81** [reply]
Magari hai già risolto. Io uso un mini pc con Debian su cui ho installato xrdp per accedere da remoto tramite Desktop Remoto da Windows. In questo modo non ho bisogno di essere loggato su Debian, mi loggo da remoto all'occorrenza.
Detto questo di fatto uso SSH il 95% delle volte.

**Reply by Dvernoc** [reply]
> Ho effettuato delle prove e per il momento ho risolto abilitando l'autologin and disabilitando il logoff. Immagino che non sia il meglio.
> 
> Cosa ne pensate? C'è la possibilita di inserire il desktop remoto come servizio all'avvio?
> 
> Inizialmnetre installando tutto il sistema operativo ho seguito anche questa guida ma non so se è aggiornata perche io VNC non lo trovato già installato.
> 
> https://indomus.it/guide/come-collegarsi-via-ssh-e-vnc-a-linux-debian/
> 
> Grazie in anticipo per il vostro supporto.

Le informazioni contenute nella guida per il collegamento via VNC non sono corrette, se non si installa un VNC Server il collegamento non è possibile.
Ho tentato l'installazione di tigervnc e poi di x11vnc sensa successo.
Qualcuno ha da indicarmi una guida affidabile?
Grazie

**Reply by 50494554524F** [reply]
usa una distribuzione con gnome 46, ora il desktop remoto è semplicissimo da configurare/usare

se sei interessato a debian, c'è da aspettare; su ubuntu funziona

**Reply by Dvernoc** [reply]
Grazie 
Ho installato Debian 12 alcuni giorni fa per poi testare Home Assistant, l'RDP da p.c. Win10 funziona, ma solo dopo aver fatto il log in.
Vorrei usare il p.c. con Debian 12 senza tastiera e monitor, peccato che l'installazione del server vnc non sia semplice come in altre distribuzioni.
Ritenterò con x11vnc.

**Reply by array81** [reply]
Non sono un esperto di Linux. Ma ho una mini PC con Debian 12 e Home Assistant (via Docker) e  i vari tools (zigbee2mqtt, mosquito, ...). Il mini PC non è collegato a tastiera e schermo e vi accedo da Windows tramite "Connessione Desktop Remoto". Ovviamente quando vi accedo devo loggarmi in Linux (tramite Desktop Remoto). Per fare ciò ho semplicemente installato xrdp su Debian. Ho notato che se uso "Desktop Remoto" su Windows ho problemi mentre se uso "Connessione Desktop Remoto" funziona. Non è quello che stai cercando di fare tu?

**Comment by 50494554524F** [comment]
gnome 46 dovrebbe portare le modifiche affinché il remote desktop funzioni come su windows... per adesso ci si deve arrangiare come fai ora: autologin + dummy hdmi

**Reply by 50494554524F** [reply]
prima di provare e perdere tempo, chiedo se qualcuno ha già provato... stavo pensando anche di fare al contrario: proxmox e mettere desktop gnome direttamente in vm e poi fare remote desktop della vm; in questo modo si potrebbero inserire le pwd e non usare l'autologin. si può fare?! un po' come si fa con virtualbox su windows insomma...

**Reply by 50494554524F** [reply]
provato proxmox; velocissimo, ma al momento è troppa roba che non mi serve... allora andrò direttamente su debian/ubuntu senza gui + cockpit + vm (distro gnome o haos che sia usando cockpit per remotizzarla)... era ovvissimo, ma sto studiando/pensando/provando senza copiare :D
